app/src/lib/git/fetch.ts

```diff
@@ -128,3 +129,44 @@ export async function fetchRefspec(
 
   await git(args, repository.path, 'fetchRefspec', options)
 }
+
+export async function fastForwardBranches(
+  repository: Repository,
+  branches: ReadonlyArray<ITrackingBranch>
+): Promise<void> {
+  if (branches.length === 0) {
+    return
+  }
+
+  const refPairs = branches.map(branch => `${branch.upstreamRef}:${branch.ref}`)
+
+  const opts: IGitExecutionOptions = {
+    successExitCodes: new Set([0, 1]),
```

Comment:
Could we add a small note about this for the future reader? Perhaps something like
```suggestion
    // Fetch exits with an exit code of 1 if one or more refs failed to update
    // which is what we expect will happen
    successExitCodes: new Set([0, 1]),
```